packages/core/admin/admin/src/content-manager/pages/ComponentSetttingsView/index.js

```diff
@@ -1,64 +1,66 @@
-import React, { memo, useEffect, useMemo, useReducer } from 'react';
-import { useParams } from 'react-router-dom';
-import { CheckPagePermissions, LoadingIndicatorPage } from '@strapi/helper-plugin';
-import { useSelector, shallowEqual } from 'react-redux';
-import axios from 'axios';
-import { axiosInstance } from '../../../core/utils';
-import { getRequestUrl, mergeMetasWithSchema } from '../../utils';
-import { makeSelectModelAndComponentSchemas } from '../App/selectors';
-import permissions from '../../../permissions';
-import crudReducer, { crudInitialState } from '../../sharedReducers/crudReducer/reducer';
-import { getData, getDataSucceeded } from '../../sharedReducers/crudReducer/actions';
-import EditSettingsView from '../EditSettingsView';
+// import React, { memo, useEffect, useMemo, useReducer } from 'react';
```

Comment:
Why is this one commented? Should we remove it or uncomment?

Reply:
Right now I have commented all the unmigrated views. I'd rather keep it like this for the moment.